com.valvesoftware.Steam.yml

```
@@ -140,14 +140,14 @@ modules:
         install -Dm644 -t /app/share/metainfo com.valvesoftware.Steam.metainfo.xml
```

Looks like Steam now installs .desktop file to `$(prefix)/lib/steam/steam.desktop`, while `$(prefix)/share/applications/steam.desktop` is an absolute symlink. We can fix build by making this symlink relative:
```suggestion
        install -Dm644 -t /app/share/metainfo com.valvesoftware.Steam.metainfo.xml
        ln -sfr $(readlink -f /app/share/applications/steam.desktop) /app/share/applications/steam.desktop
```

---

I'm not sure I totally understand the problem still but we can try that.

---

To my understanding, flatpak-builder edits .desktop file outside the sandbox, so from it's point of view `share/applications/steam.desktop` is a broken symlink, since it points to absolute path `/app/lib/steam/steam.desktop`.